opm/autodiff/StandardWellsDense.hpp

```diff
@@ -69,7 +72,9 @@ enum WellVariablePositions {
         template<typename FluidSystem, typename BlackoilIndices>
```

Comment:
Why not just pass BlackoilModelEbos as  the only template parameter, instead? No property system in here any more...